src/client/dfs/duns.c

```diff
@@ -500,6 +506,10 @@ duns_create_path(daos_handle_t poh, const char *path, struct duns_attr_t *attrp)
 			return -DER_INVAL;
 		}
 
+		if (fs.f_type == FUSE_SUPER_MAGIC) {
```

Comment:
where is that set btw? is it by fuse itself?

Reply:
It's set by the fuse kernel module.

Reply:
i see the value is used in daos_hdlr.c too.
could we have a typedef in 1 place and use that?